02-Calling-an-API/package.json

```diff
@@ -18,7 +18,7 @@
     "morgan": "^1.9.1"
   },
   "devDependencies": {
-    "@auth0/auth0-spa-js": "1.0.0-beta.2",
+    "@auth0/auth0-spa-js": "^1.0.1",
```

Comment:
shouldn't this be a dependency instead of devDependency?

Reply:
@luisrudge Possibly. I've tended to put them in dev dependencies because:

* in development mode you're going to install all the dependencies anyway
* for production, you're likely to do a `build`, in which case all the dependencies are bundled anyway.

I've just left the stuff that's needed for the server to work in `dependencies`, but I'm not overly precious about it. If you think we should rearrange the deps, let me know 👍